skf/markdown/code_examples/java/31-code_example--Single_input_validation_control--.md

```diff
@@ -0,0 +1,118 @@
+Single input validation control 
+--------------------------------
+
+***Example:***
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+package com.edw;
+
+import java.time.LocalDateTime;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class inputvalidation {
```

Comment:
This needs to be InputValidation and needs to be changed also on places where it's referenced.
Thanks.